src/cdk-experimental/menu/menu-item.ts

```diff
@@ -64,14 +64,17 @@ export class CdkMenuItem implements FocusableOption, FocusableElement, Toggler,
    * The text used to locate this item during menu typeahead. If not specified,
    * the `textContent` of the item will be used.
    */
-  @Input() typeahead: string;
+  @Input('cdkMenuItemTypeahead') typeahead: string;
```

Comment:
Should this be something like `cdkMenuItemTypeaheadLabel` so it's more explicit that this only controls the label? Also maybe the type should be nullable so it's clearer that it's empty by default.